### DISCUSSION TITLE
Suggestion: add nearform/lyra to the benchmarks

### DISCUSSION THREAD

**Discussion by paulocoghi** [nextapps-de/flexsearch#338] [Ideas]
I discovered [nearform/lyra](https://github.com/nearform/lyra) which launched just a few weeks ago and presents itself as a "fast, in-memory, typo-tolerant, full-text search engine", with lookups that takes [microseconds](https://github.com/nearform/lyra/tree/main/packages/benchmarks).

Would be interesting to see how Lyra compares to Flexsearch.

**Comment by ts-thomas** [comment]
Lyra is now added. Thanks for this suggestion.

**Comment by paulocoghi** [comment]
Thank you Thomas!
